scalableminds/webknossos
# Fix dragging annotation into a different orga’s dataset

When drag’n’dropping an annotation into a dataset in view-mode, the annotation is uploaded into that dataset, regardless of what dataset is reference din the NML tags.

This PR now makes this available if the organization name has changed as well.

### URL of deployed dev instance (used for testing):
- https://uploadannotationorgaspecific.webknossos.xyz

### Steps to test:
- Create annotation, download
- In the NML file, edit the organization name in the experiment tag to some non-existing orga, or remove it completely
- Uploading that annotation by dragging it into a dataset view mode should still work, and create an annotation for that dataset.

### Issues:
- fixes #7681

------
